### ISSUE TITLE
Dependencies not installed

### ISSUE THREAD

**Issue by roflc0pter** [dushyantb/logger-es6#1]
I get `cannot find module 'x'` every time and I don't really want to install all dependencies manually. 

```
module.js:472
    throw err;
    ^

Error: Cannot find module 'archiver-promise'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (.../node_modules/node-logger-es6/distribution/index.js:9:24)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by roflc0pter** [comment]
Node.js version: v7.10.0
NPM: 4.2.0

**Comment by bountyC0d3r** [comment]
Apologies that you have to face the issue, the issue has been fixed now. 
Please open another issue, if you still face this or any other issues.
